build.js

```diff
@@ -50,364 +50,641 @@ var androidVersion = require('./package.json').version;
+
+                gradleBuildFileData = replaceTextBetween(gradleBuildFileData, XML_START_PLUGINS_BULK_DEPENDENCIES, XML_END_PLUGINS_BULK_DEPENDENCIES, gradleJarDependencyStr);
+                return fs.writeFileAsync(gradleBuildFile, gradleBuildFileData, 'utf-8');
+            }
+            else{
+                return Promise.resolve("Success");
+            }
+        })
+}
+
 //// Utilities
 
-function transformXSL(api, inFile, outFile, xslFile, params) {
-  for (var key in params) {
-    if (typeof params[key] !== 'string') {
-      if (!params[key] || typeof params[key] === 'object') {
-        logger.error("settings for AndroidManifest: value for", chalk.yellow(key), "is not a string");
-      }
+function transformXSL(api, inFile, outFile, xslFile, params, config) {
+    for (var key in params) {
+        if (typeof params[key] !== 'string') {
+            if (!params[key] || typeof params[key] === 'object') {
+                logger.error("settings for AndroidManifest: value for", chalk.yellow(key), "is not a string");
+            }
+
 
-      params[key] = JSON.stringify(params[key]);
+            params[key] = JSON.stringify(params[key]);
+        }
     }
-  }
-
-  var outFileTemp = outFile + ".temp";
-  return new Promise(function (resolve, reject) {
-      api.jvmtools.exec({
-        tool: 'xslt',
-        args: [
-          "--in", inFile,
-          "--out", outFileTemp,
-          "--stylesheet", xslFile,
-          "--params", JSON.stringify(params)
-          ]
+
+    params['package'] = config.packageName
+
+    var outFileTemp = outFile + ".temp";
+    return new Promise(function (resolve, reject) {
+        api.jvmtools.exec({
+            tool: 'xslt',
+            args: [
+                "--in", inFile,
+                "--out", outFileTemp,
+                "--stylesheet", xslFile,
+                "--params", JSON.stringify(params)
+            ]
         }, function (err, xslt) {
-          if (err) { return reject(err); }
+            if (err) { return reject(err); }
 
-          var logger = api.logging.get('xslt');
-          xslt.on('out', logger.out);
-          xslt.on('err', logger.err);
-          xslt.on('end', resolve);
+            var logger = api.logging.get('xslt');
+            xslt.on('out', logger.out);
+            xslt.on('err', logger.err);
+            xslt.on('end', resolve);
         });
     })
-    .then(function () {
-      return fs.readFileAsync(outFileTemp, 'utf-8');
-    })
-    .then(function(contents) {
-      contents = contents.replace(/android:label=\"[^\"]*\"/g, "android:label=\""+params.title+"\"");
-      fs.writeFileAsync(outFile, contents, 'utf-8');
-    });
+        .then(function () {
+            return fs.readFileAsync(outFileTemp, 'utf-8');
+        })
+        .then(function(contents) {
+            //contents = contents.replace(/android:label=\"[^\"]*\"/g, "android:label=\""+params.title+"\"");
```

Comment:
If this is not used, we should remove it

Reply:
Done